extensions/wikia/ImageServing/drivers/ImageServingDriverMainNS.class.php

```diff
@@ -160,74 +161,55 @@ protected function loadImageDetails( $imageNames = array() ) {
 		wfProfileOut( __METHOD__ );
 	}
 
-	protected function getImagesPopularity( $imageNames, $limit ) {
-		global $wgContentNamespaces;
-
+	/**
+	 * Returns the popularity of given set of images (popularity = the number of articles in content namespaces that use an image)
+	 *
+	 * $limit is applied - images that have popularity that is higher than $limit will not be returned
+	 *
+	 * Example:
+	 *
+	 * $imageNames = [ 'IMG_7303.jpg', 'Koziołki.jpg' ]
```

Comment:
Let's keep the examples in English unless they are language-specific. Imagine yourself seeing a Chinese text in phpdoc ;-)